src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -138,6 +138,13 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// </summary>
         public bool StartFromBeginning { get; set; } = false;
 
+        /// <summary>
+        /// Optional.
+        /// Gets or sets the DateTime from which feed should start from. 
```

Comment:
DateTime capitalized is related to C#, but the trigger is used in other languages.

```suggestion
        /// Gets or sets the a date and time to initialize the change feed read operation from.
```